README.md

```diff
@@ -1,24 +1,32 @@
-                               Autopsy 3.0
-                          http://www.sleuthkit.org/
+#                               Autopsy 4.0
+http://www.sleuthkit.org/
 
-                                March 13, 2013
+March 3, 2016
 
 
-OVERVIEW
+
+##OVERVIEW
 
 Autopsy is a graphical interface to The Sleuth Kit and other open source digital forensics tools. 
-Autopsy 3 is a complete rewrite from Autopsy 2 and it is now Java-based.     
+Autopsy 4 (and 3) are a complete rewrite from Autopsy 2, and none of this document is relevant to Autopsy 2.   
 
```

Comment:
I would prefer to say:

Autopsy 4 improves on Autopsy 3 by supporting collaboration on a single case by multiple users. Autopsy 3 was a complete rewrite of Autopsy 2 using Java, and the information presented in this document is not relevant to Autopsy 2.